app/celery/provider_tasks.py

```diff
@@ -31,10 +33,10 @@ def deliver_sms(self, notification_id):
             )
             self.retry(queue=QueueNames.RETRY)
         except self.MaxRetriesExceededError:
-            current_app.logger.exception(
-                "RETRY FAILED: task send_sms_to_provider failed for notification {}".format(notification_id),
-            )
-            update_notification_status_by_id(notification_id, 'technical-failure')
+            message = "RETRY FAILED: task send_sms_to_provider failed for notification {}. " \
```

Comment:
Should this message message that its a Max Retries Failure not just a Retry Failure?

Reply:
Perhaps - but if the message is marked as technical failure than it has retried the max number of times.  (I'll add max to the message)